sensor_msgs/JointState joint_goal
---
bool finished
